cmds/modules/netlightd/nft/rules.nft

```diff
@@ -0,0 +1,29 @@
+add table inet filter;
```

Comment:
What I meant with this is to make it similar to other `rules.nft` (in the same PR here style. It doesn't have commands but instead the complete structure of the rule set

It contains simply the output of running `nft list ruleset` 

If you don't get what  I mean let me explain over DM